cmd/get.go

```diff
@@ -89,20 +89,20 @@ func (o *GetOptions) Complete(f Factory, args []string) (err error) {
 	}
 
 	if o.Selector == "body" {
-		// if we have a limit, but not offset, assume an offset of 0
-		if o.Limit != -1 && o.Offset == -1 {
-			o.Offset = 0
+		// if we have a PageSize, but not Page, assume an Page of 1
+		if o.PageSize != -1 && o.Page == -1 {
+			o.Page = 1
```

Comment:
Do pages start being indexed at 1 instead of 0?

Reply:
yes!